cmds/exp/pox/pox.go

```diff
@@ -94,31 +104,37 @@ import (
 
 const usage = "pox [-[-debug]|d] -[-run|r] | -[-create]|c  [-[-file]|f tcz-file] file [...file]"
 
+type mp struct {
+	source string
+	target string
+	fstype string
+	flags  uintptr
+	data   string
+	perm   os.FileMode // for target in the chroot
+}
+
 var (
 	debug  = flag.BoolP("debug", "d", false, "enable debug prints")
 	run    = flag.BoolP("run", "r", false, "Run the first file argument")
 	create = flag.BoolP("create", "c", false, "create it")
 	zip    = flag.BoolP("zip", "z", false, "use zip instead of squashfs")
 	file   = flag.StringP("output", "f", "/tmp/pox.tcz", "Output file")
-	v      = func(string, ...interface{}) {}
+	extra  = flag.StringP("extra", "e", "", `comma-separated list of extra directories to add (on create) and binds to do (on run).
+You can specify what directories to add, and when you run, specify what directories are bound over them, e.g.:
+pox -c -e /tmp,/etc commands ....
+pox -r -e /a/b/c/tmp:/tmp,/etc:/etc commands ...
```

Comment:
what happens if you do both create and run with -e?  i get that the -e arguments with -c are the directories you create in the pox.  on -r, the directories are host_dir:pox_dir mounts.  but what about both -c -r?

Reply:
ah, after reading the code, i see the default is /dir:/dir, such that whatever comes after the : is the alternate name inside the chroot.  and if you just have a single name (no : ), then it's the same name for -c and -r.

the way your example worked, i thought you had to do the : style for -r.

maybe put this info under the examples section?